### ISSUE TITLE
iOS Crash on Swift 5

### ISSUE THREAD

**Issue by nithinnizam** [clostra/newnode#87]
Trying to integrate newnode SDK into iOS application created with Swift5. Tried the following method described in the Readme file. 

Added via Cocoapods.

`pod 'NewNode'
`
Created a session object. 

```
let sessionConfig = URLSessionConfiguration.default
sessionConfig.connectionProxyDictionary = NewNode.connectionProxyDictionary // Crashing
let session = URLSession(configuration: sessionConfig)

```

**Comment by bteitelb** [comment]
@nithinnizam sorry you are having trouble with this. It should work. What is reported when it crashes? If you turn on logging, do you see any information in the log that could help us figure out why it's crashing for you?

**Comment by nithinnizam** [comment]
@bteitelb : We are not able to turn on logging. It crashes with EXC_BAD_ACCESS if we try to access any method/object in the Framework.

**Comment by nithinnizam** [comment]
Please see the demo project we setup to show you the issue that we are facing, 
https://github.com/nithinnizam/newnodedemo-ios

**Comment by ghazel** [comment]
The issue here is that NewNode looks for `CFBundleDisplayName` in the Info.plist. This sample only has `CFBundleName` set. I added fallback code in a3c58e0ccc9e62a8942c82a6a1558efdf366f5ee which will be included in NewNode 1.8.1. A workaround is to set `CFBundleDisplayName` manually in your project.

**Comment by nithinnizam** [comment]
Got the issue fixed in the latest release, thanks for the response.
